crates/typst-library/src/compute/construct.rs

```diff
@@ -754,6 +754,35 @@ pub fn range(
     Ok(array)
 }
 
+/// Loads a wasm plugin and return a plugin instance.
+///
+/// Plugins functions may accept strings as argument and
+/// return a single string.
+///
+/// ## Example { #example }
+/// ```example
+/// #let plugin = plugin("hello.wasm")
+///
+/// #plugin.hello() // returns the string hello world
```

Comment:
The preview will show that it returns "hello world" :)
```suggestion
/// #let plugin = plugin("hello.wasm")
/// #plugin.hello() 
```